files/slave.dockerfile

```diff
@@ -2,21 +2,20 @@ FROM docker.io/stackbrew/centos:latest
 MAINTAINER Chris Evich <cevich@redhat.com>
 
 RUN yum update -y && \
-    yum install -y findutils epel-release https://rdoproject.org/repos/rdo-release.rpm && \
+    yum install -y epel-release https://rdoproject.org/repos/rdo-release.rpm && \
```

Comment:
Why are you removing findutils? I wonder if this is what's making `run_exec/exec_pid_count` fail on fedora?